Documentation/reference-guides/xfrm/index.rst

```diff
@@ -136,10 +136,10 @@ the mark, the SPI, and the protocol. In case of any decode error (ex., wrong key
 dropped and an error counter is increased.
 
 As illustrated on the diagram, an XFRM policy matching the packet isn't required for the decoding
-to happen (it goes directly to "XFRM decode"), but it may be required for the packet to proceed to
-a local process or through the FORWARD chain. An XFRM policy with an optional template (i.e.,
-``level use``) is also required if no state or socket matches it (ex., userspace proxy redirects in
-Cilium).
+to happen (it goes directly to "XFRM decode"), but it is required for the packet to proceed to a
```

Comment:
Tiny nit, it reads better if you drop the 'it' in this line. The 'is' automatically references the previous subject.

Reply:
I'm always confused about this one. I know it's common to skip the subject in the "but" clauses, but AFAIU, those clauses are actually independent so I'd expect they would have a subject :shrug: 

Anyway, I removed it.